qmlti/src/resources/PerceptionSoap.php

```diff
@@ -317,6 +316,72 @@ public function get_access_assessment_notify($assessment_id, $participant_name,
     return $access_assessment;
   }
 
+  public function create_schedule_participant($schedule_id, $schedule_name, $assessment_id, $participant_id, $restrict_times = TRUE, $schedule_starts, $schedule_stops, $group_id, $group_tree_id, $web_delivery, $restrict_attempts, $max_attempts, $monitored, $test_center_id, $min_days_between_attempts, $time_limit_override, $time_limit, $offline_delivery) {
+    try {
+      $access_parameters = array(
+        "schedule" => array(
+          "Schedule_ID" => $schedule_id,
+          "Schedule_Name" => $schedule_name,
+          "Assessment_ID" => $assessment_id,
+          "Participant_ID" => $participant_id,
+          "Restrict_Times" => $restrict_times,
+          "Schedule_Starts" => $schedule_starts,
+          "Schedule_Stops" => $schedule_stops,
+          "Group_ID" => $group_id,
+          "Group_Tree_ID" => $group_tree_id,
+          "Web_Delivery" => $web_delivery,
+          "Restrict_Attempts" => $restrict_attempts,
+          "Max_Attempts" => $max_attempts,
+          "Monitored" => $monitored,
+          "Test_Center_ID" => $test_center_id,
+          "Min_Days_Between_Attempts" => $min_days_between_attempts,
+          "Time_Limit_Override" => $time_limit_override,
+          "Time_Limit" => $time_limit,
+          "Offline_Delivery" => $offline_delivery
+        )
+      );
+      $schedule_id = $this->soap->CreateScheduleParticipantV42($access_parameters);
+    } catch(SoapFault $e) {
+      throw new QMWiseException($e);
+    }
+    return $schedule_id;
+  }
+
+
+  public function get_access_schedule_notify($schedule_id, $participant_name, $consumer_key, $resource_link_id, $result_id, $notify_url, $home_url, $participant_id, $additional_params = array()) {
```

Comment:
Missing PHPDoc. Probably a good idea if someone else has to maintain this in the future.